Admin/Emails/Emails.rst

```diff
@@ -45,6 +45,9 @@ After you have configured the Mail Account Settings, be sure to enable fetching
 .. image:: ../../_static/images/admin_emails_emails_enable_fetching.png
   :alt: Enable Fetching
 
+**Fetch From Mail Folder:** This feature adds a textbox in the Email configurations that allows you to configure a specific Folder to fetch from. If no folder is provided the system will fetch from the INBOX as usual. This also adds a check so that if a Folder is given but doesn't exist and can't be created then the system will error out with a detailed message.
```

Comment:
The following line needs to be removed:
>This also adds a check so that if a Folder is given but doesn't exist and can't be created then the system will error out with a detailed message.

This is a dev comment and is not meant for Documentation.